packages/axiom-components/src/Dropdown/Dropdown.js

```diff
@@ -59,6 +59,17 @@ export default class Dropdown extends Component {
     this.state = { isVisible: false };
   }
 
+  componentDidUpdate() {
+    const { enabled, onRequestClose } = this.props;
+    const { isVisible } = this.state;
+
+    if (!enabled && isVisible) {
+      this.setState({ isVisible: false });
+
+      if (onRequestClose) onRequestClose();
```

Comment:
I'm not sure if there is a linter in place to check but I think this is against the code-style
```suggestion
      if (onRequestClose) {
        onRequestClose();
      }
```

Reply:
Fair enough, I thought I get way with it since the linter allows it 😄

Reply:
@stefanpearson `¯\_(ツ)_/¯`

![image](https://user-images.githubusercontent.com/423234/156358174-bc7b59d3-131b-4947-bf83-c411fb93d234.png)